### ISSUE TITLE
Labels of form field buttons always centered

### ISSUE THREAD

**Issue by manuel-192** [v1cont/yad#122]
Yad version 8.0.

`yad --form --width=400 --align=left --field="Firefox":fbtn "firefox"`

Yad man page says form field label is by default left aligned, so the `--align` option above should not even be needed.
But the code above will show center aligned label `Firefox`. And form option `--align=TYPE` with any TYPE value does not change anything.

![yad](https://user-images.githubusercontent.com/32041133/108399793-2ed40200-7223-11eb-92a0-f479799940a2.png)

**Comment by Misko-2083** [comment]
@v1cont It's doable with an GtkOverlay.
I had some spare time to make a GtkApp example in C:
https://gist.github.com/Misko-2083/6621bb5ddbcdf18f17be00979d81f704
Compliles with:
```bash
gcc -o button_label -fPIC button_label.c `pkg-config --cflags --libs gtk+-3.0`
```
In action on Debian Buster:
![peekMSJ1Z0](https://user-images.githubusercontent.com/8826552/111044459-ec16db80-8416-11eb-9057-97ef0eed6caa.gif)
Without the dummy label in the button `widgets->button = gtk_button_new_with_label("");` it looks impossible to set the right button height when the font size setting is changed in the settings.

**Comment by manuel-192** [comment]
Interesting! Good job!

**Comment by v1cont** [comment]
fixed in a29cade with additional option --align-buttons
